src/sentry/static/sentry/app/actionCreators/incident.jsx

```diff
@@ -46,7 +46,9 @@ export async function createIncident(
 /**
  * Fetches a list of activities for an incident
  */
-export async function fetchIncidentActivities(api, orgId, incidentId) {}
+export async function fetchIncidentActivities(api, orgId, incidentId) {
```

Comment:
Should all of our API actionCreators be async functions like this?

Reply:
In an ideal world components wouldn't be able to listen to them (instead they'd have to only listen to a store) - I made it async here to be consistent with the rest (which `await` to display status messages).